data/examples/shaders/transition/luma-burn.effect

```diff
@@ -145,11 +154,176 @@ float4 PSInverse(VertData v_in) : TARGET {
 	//return transition;
 }
 
-technique DrawInverse
+technique DrawLightToDark
```

Comment:
Same here.

Reply:
I put at the beginning of the techniques